source/includes/_tipser-tools.md

```diff
@@ -1,3 +1,35 @@
 # Tipser Tools
 
-//tools desc
\ No newline at end of file
+Tipser enables commerce on almost any site. An easy way to enable shopping on your site is to install the Tipser script and use the Tipser Tools to insert `Get The Look` widget directly into your content. The installation of the script on your site typically takes less than 5 minutes, and can be done by even non-technical people. 
```

Comment:
I'd suggest something like this: `to insert `Get The Look` widget directly into your content` -> `to insert product listings and buy buttons directly into your site`